src/object.d

```diff
@@ -1641,6 +1643,11 @@ class Throwable : Object
         string toString() const;
     }
 
+    private uint _refcount;     // 0 : allocated by GC
```

Comment:
I thought about this more and realized that in order to do this, you would have to create 2 vtables for EVERY CLASS that inherits from throwable, one for RC objects and one for GC objects.  A solution like this could be worth looking into for a general solution, however, I don't think is appropriate for this feature.